unittests/allplatformstests.py

```diff
@@ -4086,6 +4088,10 @@ def output_name(name, type_):
                     'destination': '{bindir}/' + exe_name('app'),
                     'tag': 'runtime',
                 },
+                f'{self.builddir}/' + exe_name('app-otherdir'): {
+                    'destination': '{prefix}/otherbin/' + exe_name('app-otherdir'),
```

Comment:
We want this to be `{bindir}`.

Reply:
Notice that it's getting installed to "otherbin", which is a custom `install_dir`.